packages/synapse-react-client/src/components/StandaloneQueryWrapper/StandaloneQueryWrapper.tsx

```diff
@@ -105,10 +105,8 @@ const StandaloneQueryWrapper: React.FunctionComponent<
 
   const derivedQueryRequestFromSearchParams = generateInitQueryRequest(sql)
 
-  if (searchParams) {
```

Comment:
Removed check of searchParams, since this is done by getAdditionalFilters().  getAdditionalFilters() will return undefined if there are no search params and nothing found in local storage.